go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -118,7 +118,7 @@ func (vh *vtgateHandler) ConnectionClosed(c *mysql.Conn) {
 }
 
 // Regexp to extract parent span id over the sql query
-var r = regexp.MustCompile("/\\*VT_SPAN_CONTEXT=(.*)\\*/")
+var r = regexp.MustCompile(`/\\*VT_SPAN_CONTEXT=(.*)\\*/`)
```

Comment:
Nice find! The un-escaped string should be:
```
`/\*VT_SPAN_CONTEXT=(.*)\*/`
```

Reply:
OK Thank you